.github/workflows/actions.yml

```diff
@@ -12,6 +12,8 @@ on:
     - >-
       **
   pull_request:
+  schedule:
```

Comment:
You're just introducing a daily CI build here, right?

Reply:
Exactly.

Reply:
Thank you for clarifying!